Recrutify/Recrutify.Host/Controllers/OData/CandidatesController.cs

```diff
@@ -23,7 +23,7 @@ public CandidatesController(ICandidateService candidateService)
 
         [EnableQuery(
             HandleNullPropagation = HandleNullPropagationOption.False,
-            MaxAnyAllExpressionDepth = 2,
+            MaxAnyAllExpressionDepth = 3,
```

Comment:
Why 3? It'a not necessary if it doesn't help with sorting

Reply:
done